.github/workflows/ci.yaml

```diff
@@ -1,5 +1,9 @@
 name: CI
-on: push
+on: 
+  push:
+    branches: [master]
```

Comment:
This still isn't good solution... If I am working on something I want to run `ci`, even if PR is not opened. I was thinking about something more like ` if: ${{ github.event_name == 'pull_request' && github.event.pull_request.head.repo.full_name != github.repository }}`